assertsprocessor/config_refresh.go

```diff
@@ -0,0 +1,85 @@
+package assertsprocessor
+
+import (
+	"encoding/json"
+	"github.com/tilinna/clock"
+	"go.uber.org/zap"
+	"net/http"
+)
+
+type configListener interface {
+	isUpdated(prevConfig *Config, currentConfig *Config) bool
```

Comment:
`currConfig` and `newConfig` sounds more intuitive to me

Reply:
Sure, will modify this. I was a bit confused on the naming of this one, but `currConfig` and `newConfig`  indeed does sound more intuitive